client/components/breadcrumb/README.md

```diff
@@ -31,4 +31,4 @@ const BreadcrumbExamples = () => {
 
 - `items` (`{ label: string; href?: string; helpBubble?: React.ReactElement; onClick?: () => void }[]`) - The Navigations items to be shown
 - `compact` (`boolean`) - Displays only the previous item URL (optional)
-- `mobileItem` (`string`) - In compact version, displays this value. If not passed defaults to "Back" (optional)
+- `mobileItem` (`{ label: string; href?: string; helpBubble?: React.ReactElement; onClick?: () => void }`) - In compact version, displays this value. If not passed defaults to "Back" and the href of the second-to-last item (optional)
```

Comment:
I made this change to the README because I noticed that the `mobileItem` prop type is `Item`, not string.